src/app/@ansyn/ansyn/modules/menu-items/cases/models/case.model.ts

```diff
@@ -41,8 +44,8 @@ export interface IImageManualProcessArgs {
 	Sharpness?: number;
 }
 
-export interface IOverlaysManualProcessArgs {
-	[key: string]: IImageManualProcessArgs;
+export interface IOverlaysImageProcess {
+	[key: string]: IOverlayImageProcess;
```

Comment:
can you explain this syntax: [key: string]

Reply:
is mean that the key could be any string and the value must be `IOverlayImageProcess`